flathunter/abstract_crawler.py

```diff
@@ -75,7 +49,8 @@ def configure_driver(self, driver_path, driver_arguments):
 
     def rotate_user_agent(self):
         """Choose a new random user agent"""
-        self.HEADERS['User-Agent'] = random.choice(self.USER_AGENTS)
+        user_agent_rotator = UserAgent()
```

Comment:
In my local tests, it seems like the user_agent_rotator is sometimes returning Mobile Device UA strings, which makes some of the websites redirect to their app / mobile version. Looking at the docs, it seems like we need:

```
from random_user_agent.params import HardwareType, Popularity

user_agent_rotator = UserAgent(popularity=[Popularity.COMMON], hardware_types=[HardwareType.COMPUTER])
```

to avoid those problems.